src/gpio.rs

```diff
@@ -199,7 +199,10 @@ macro_rules! gpio {
 
             impl<MODE> ExtiPin for $PXx<Input<MODE>> {
                 /// Make corresponding EXTI line sensitive to this pin
-                fn make_interrupt_source(&mut self, syscfg: &mut SYSCFG) {
+                fn make_interrupt_source(&mut self, syscfg: &mut SYSCFG, rcc: &mut RCC) {
+                    // SYSCFG clock must be enabled in order to do register writes
+                    rcc.apb2enr.modify(|_, w| w.syscfgen().set_bit());
```

Comment:
> However since I'm a bit new to this crate I'm unsure what the best approach is (support in gpio.rs, rcc.rs?).

This looks good to me atm despite it being a slightly different api than the F4 hal for instance. From the reference manual, the SYSCFG is only really used for a few things on the F7 (remap memory, select ethernet phy, exti). I could see maybe moving this to `rcc.rs` when more features are enabled, but seems ok here for now.